api/api_sources/schema-migration-sql/CountryProvinceSchema/CountryProvinceSchema.sql

```diff
@@ -0,0 +1,37 @@
+-- ### Creating Table: country_province ### --
+
+        
+CREATE TABLE country_province ();
+ALTER TABLE country_province ADD COLUMN province_code VARCHAR(2) NOT NULL;
+ALTER TABLE country_province ADD COLUMN country_code VARCHAR(3) NOT NULL;
+ALTER TABLE country_province ADD COLUMN description VARCHAR(100) NULL;
```

Comment:
The "description" column should be **NOT NULL** as there should always be a description.

Reply:
Resolved